### PR TITLE
Add hatgoos tutorial

### PR DESCRIPTION
**Pull request by Tatohead**
Not sure how this works but you can edit this if needed. This is the tutorial for one of the mods

### CODE REVIEW

**Comment by VukAnd** [comment]
Could you add this to the hatgoos.md file inside mods instead? Looks kinda cluttered on the front page.

File: README.md [hunk 1]
```diff
@@ -25,3 +25,58 @@
 
 ## Tutorials and info
 * [Desktop Goose Modding Tutorial](https://www.youtube.com/watch?v=5OQiVWJ-TGc) by iwanMods (external link)
+### Hatgoos Tutorial
+Author of this mod is DaNike#6223
+
+This mod allows you to have your goose wear hats it can also be run with other mods
+
+<br/>
```
Comment (Wuzado) [inline README.md]: Not needed here + avoid using HTML

File: README.md [hunk 2]
```diff
@@ -25,3 +25,58 @@
...
+• First download the hatgoos mod
+
+The file should say "hatgoos_with_loader.zip" when you download it
+
+• Make sure to load it in the same area as the DesktopGoose.exe to unzip it should something like this
+https://cdn.discordapp.com/attachments/672263146240737280/673282204100460554/unknown.png
+
+<br/>
```
Comment (Wuzado) [inline README.md]: Same as line 33

File: README.md [hunk 3]
```diff
@@ -25,3 +25,58 @@
...
+• Then run the 'run with mods.bat' that you see above
+
+• Then exit that program
+
+• Open up the newly created config file
+
+This is what you should kinda see (or something similar, as long it has the last 4 lines the same on that notepad)
+https://cdn.discordapp.com/attachments/672263146240737280/673283260335521812/unknown.png
+
+<br/>
```
Comment (Wuzado) [inline README.md]: Same as line 33

File: README.md [hunk 4]
```diff
@@ -25,3 +25,58 @@
...
+• Edit 
+HatGoos.HatMode=Default
+
+• To this
+HatGoos.HatMode=Custom
+
+<br/>
```
Comment (Wuzado) [inline README.md]: Same as line 33

File: README.md [hunk 5]
```diff
@@ -25,3 +25,58 @@
...
+• Last but not least copy the location of the hat you want which you can do by clicking the search bar of your file manager and copying it.
+
+See here:
+https://cdn.discordapp.com/attachments/672554163925942328/673297535464505381/unknown.png
+
+Note: Don't forget to include the name of the image file
+
+• Paste it in HatGoos.CustomHatPath= 
+
+<br/>
```
Comment (Wuzado) [inline README.md]: Same as line 33

File: README.md [hunk 6]
```diff
@@ -25,3 +25,58 @@
...
+So it should look more like this
+
+Example:
+HatGoos.CustomHatPath=C:\Users\user\Downloads\PC\Desktop Goose\Assets\Images\Hats\hat5.png
+
+<br/>
```
Comment (Wuzado) [inline README.md]: Same as line 33

File: README.md [hunk 7]
```diff
@@ -25,3 +25,58 @@
...
+• Remember to save the Config file's changes
+
+• Run the 'run with mods.bat' file once more and you're done!
+
+<br/>
```
Comment (Wuzado) [inline README.md]: Same as line 33

File: README.md [hunk 8]
```diff
@@ -25,3 +25,58 @@
 
 ## Tutorials and info
 * [Desktop Goose Modding Tutorial](https://www.youtube.com/watch?v=5OQiVWJ-TGc) by iwanMods (external link)
+### Hatgoos Tutorial
+Author of this mod is DaNike#6223
+
+This mod allows you to have your goose wear hats it can also be run with other mods
+
+<br/>
+• First download the hatgoos mod
```
Comment (Wuzado) [inline README.md]: Switch to using MD formatting - use asterisks or minuses.

File: README.md [hunk 9]
```diff
@@ -25,3 +25,58 @@
...
+
+The file should say "hatgoos_with_loader.zip" when you download it
+
+• Make sure to load it in the same area as the DesktopGoose.exe to unzip it should something like this
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 10]
```diff
@@ -25,3 +25,58 @@
...
+https://cdn.discordapp.com/attachments/672263146240737280/673282204100460554/unknown.png
+
+<br/>
+• Delete the config file
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 11]
```diff
@@ -25,3 +25,58 @@
...
+
+• Then run the 'run with mods.bat' that you see above
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 12]
```diff
@@ -25,3 +25,58 @@
...
+
+• Then exit that program
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 13]
```diff
@@ -25,3 +25,58 @@
...
+
+• Open up the newly created config file
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 14]
```diff
@@ -25,3 +25,58 @@
...
+
+This is what you should kinda see (or something similar, as long it has the last 4 lines the same on that notepad)
+https://cdn.discordapp.com/attachments/672263146240737280/673283260335521812/unknown.png
+
+<br/>
+• Edit
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 15]
```diff
@@ -25,3 +25,58 @@
...
+• Open up the newly created config file
+
+This is what you should kinda see (or something similar, as long it has the last 4 lines the same on that notepad)
+https://cdn.discordapp.com/attachments/672263146240737280/673283260335521812/unknown.png
+
+<br/>
+• Edit 
+HatGoos.HatMode=Default
+
+• To this
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 16]
```diff
@@ -25,3 +25,58 @@
...
+HatGoos.HatMode=Custom
+
+<br/>
+• Last but not least copy the location of the hat you want which you can do by clicking the search bar of your file manager and copying it.
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 17]
```diff
@@ -25,3 +25,58 @@
...
+
+See here:
+https://cdn.discordapp.com/attachments/672554163925942328/673297535464505381/unknown.png
+
+Note: Don't forget to include the name of the image file
+
+• Paste it in HatGoos.CustomHatPath=
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 18]
```diff
@@ -25,3 +25,58 @@
...
+• Paste it in HatGoos.CustomHatPath= 
+
+<br/>
+So it should look more like this
+
+Example:
+HatGoos.CustomHatPath=C:\Users\user\Downloads\PC\Desktop Goose\Assets\Images\Hats\hat5.png
+
+<br/>
+• Remember to save the Config file's changes
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 19]
```diff
@@ -25,3 +25,58 @@
...
+
+• Run the 'run with mods.bat' file once more and you're done!
```
Comment (Wuzado) [inline README.md]: Same as line 34

File: README.md [hunk 20]
```diff
@@ -25,3 +25,58 @@
...
+
+This mod allows you to have your goose wear hats it can also be run with other mods
+
+<br/>
+• First download the hatgoos mod
+
+The file should say "hatgoos_with_loader.zip" when you download it
+
+• Make sure to load it in the same area as the DesktopGoose.exe to unzip it should something like this
+https://cdn.discordapp.com/attachments/672263146240737280/673282204100460554/unknown.png
```
Comment (Wuzado) [inline README.md]: Use image embeds: `alt text`

File: README.md [hunk 21]
```diff
@@ -25,3 +25,58 @@
...
+• Delete the config file
+
+• Then run the 'run with mods.bat' that you see above
+
+• Then exit that program
+
+• Open up the newly created config file
+
+This is what you should kinda see (or something similar, as long it has the last 4 lines the same on that notepad)
+https://cdn.discordapp.com/attachments/672263146240737280/673283260335521812/unknown.png
```
Comment (Wuzado) [inline README.md]: Same as line 39

File: README.md [hunk 22]
```diff
@@ -25,3 +25,58 @@
...
+HatGoos.HatMode=Default
+
+• To this
+HatGoos.HatMode=Custom
+
+<br/>
+• Last but not least copy the location of the hat you want which you can do by clicking the search bar of your file manager and copying it.
+
+See here:
+https://cdn.discordapp.com/attachments/672554163925942328/673297535464505381/unknown.png
```
Comment (Wuzado) [inline README.md]: Same as line 39

**Comment by Tatohead** [comment]
Alright, I'll edit format (removing html) and move the tutorial over to the hatgoos. I'm not very familiar with GitHub, I'm sorry about that. I added the FAQ just in case with that section

**Comment by VukAnd** [comment]
There was a time when we all were unfamiliar with GitHub. :)

**Comment by cuboide** [comment]
The audacity of TypoCheck to even consider "hatgoos" is a typo.

File: README.md (hunk 1 above)
Comment (Tatohead) [reply README.md]: I removed the <br/> parts and change the image link to the correct format just now. Thank you for showing me the correct guide.

**Comment by Tatohead** [comment]
Oh okay I'll try editing it moe

**Comment by Tatohead** [comment]
Hmm... I think the alt text could be better for describing the third image but I'm not sure how to express it better.

**Comment by Tatohead** [comment]
wait did I do it wrong again?

**Pull request merged**